src/main/java/io/unlogged/logging/util/NetworkClient.java

```diff
@@ -70,29 +72,29 @@ public void sendPOSTRequest(String url, String attachmentFilePath) throws IOExce
         MultipartUtility form = null;
         try {
             form = new MultipartUtility(url, charset, headers);
-
-            File binaryFile = new File(attachmentFilePath);
-            form.addFilePart("file", binaryFile);
-            form.addFormField("sessionId", sessionId);
-            form.addFormField("hostname", getHostname());
-
-            String response = form.finish();
+            form.addFilePart("file", new File(attachmentFilePath));
+            form.addFilePart("file", new File(loggerPath));
```

Comment:
different parameter name, you already used "file"

Reply:
I think that will fail. The server takes file as an array of files (changed that flow today)